components/style/values/specified/source_size_list.rs

```diff
@@ -38,9 +41,12 @@ impl Parse for SourceSize {
 /// A value for a `<source-size-list>`:
 ///
 /// https://html.spec.whatwg.org/multipage/#source-size-list
+#[derive(Debug)]
 pub struct SourceSizeList {
-    source_sizes: Vec<SourceSize>,
-    value: Option<Length>,
+    /// https://html.spec.whatwg.org/multipage/#source-size-list
+    pub source_sizes: Vec<SourceSize>,
+    /// https://html.spec.whatwg.org/multipage/#source-size-list
+    pub value: Option<Length>,
```

Comment:
To avoid making these fields public, perhaps we can add a method called `set_fallback_value` that mutates `self.value`.